hacks/iptables/buildvm-scripts/canttouchthat.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import argparse
+import click
+import subprocess
+import shlex
+
+RULE_COMMENT = "Rule managed by ART"
+
+
+def make_output_args(cidr):
+    return ['OUTPUT',
+            '-d', cidr,
+            '-j', 'ACCEPT',
+            '-m', 'comment',
+            '--comment', RULE_COMMENT
+            ]
+
+@click.command()
+@click.option('-n', '--output-networks', metavar='FILE',
```

Comment:
Consider the following

https://click.palletsprojects.com/en/7.x/arguments/#file-arguments